src/test/shared/debug/launchConfiguration.test.ts

```diff
@@ -33,7 +33,7 @@ const debugConfigurations: vscode.DebugConfiguration[] = [
     },
 ]
 
-const templateUri = vscode.Uri.file('/')
+const templateUri = vscode.Uri.file('/template.yaml')
```

Comment:
just a cosmetic change, not related to this PR. Helpful to have intuitive values in the tests.